src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```
@@ -0,0 +1,190 @@
+            return updated_display
+        except HTTPException as exception:
+            if exception.status == StatusCodes.PRECONDITION_FAILED:
+                logger.debug(
+                    'Retrying setting %s times with exception: concurrent setting operations', i + 1)
+                time.sleep(retry_interval)
+            else:
+                raise CLIError(str(exception))
+        except Exception as exception:
+            raise CLIError(str(exception))
+    raise CLIError(
+        "Fail to set the key '{}' due to a conflicting operation.".format(key))
+
+
+def show_feature(cmd,
+            feature,
+            name=None,
+            label=None,
+            fields=None,
+            connection_string=None):     
+    key = FEATURE_FLAG_PREFIX + feature
+    content_type = FEATURE_FLAG_CONTENT_TYPE
+    connection_string = resolve_connection_string(cmd, name, connection_string)
+    azconfig_client = AzconfigClient(connection_string)
+    
+    # If user has specified fields, we still get all the fields and then filter what we need from the response. 
+    query_option = QueryKeyValueOptions(label=label, fields=None, content_type=content_type)
+    
+    try:
+        key_value = azconfig_client.get_keyvalue(key, query_option)
+        if key_value is None:
+            raise CLIError("The Feature Flag does not exist.")
+        
+        feature_flag_value = mapper.map_json_to_featureflagvalue(json.loads(key_value.value))
+        feature_flag_display = mapper.map_featureflag_value_to_display(feature_flag_value)
+
+        # locked and LastModified values come from the KeyValue object
+        feature_flag_display.locked = key_value.locked
+        feature_flag_display.last_modified = key_value.last_modified
+        
+        if fields:
+            partial_ff = {}
+            for field in fields:
+                partial_ff[field.name.lower()] = feature_flag_display.__dict__[field.name.lower()]
+            return partial_ff
```

use getattr() to get the attribute by name. You can define if error out or set a default value if attribute is not found.